test/integration/storage/StorageConfig.test.js

```diff
@@ -0,0 +1,110 @@
+const { startTracker } = require('streamr-network')
+const fetch = require('node-fetch')
+const cassandra = require('cassandra-driver')
+const ethers = require('ethers')
+const { waitForCondition } = require('streamr-test-utils')
+const { StreamMessage } = require('streamr-network').Protocol.MessageLayer
+
+const { startBroker, createClient } = require('../../utils')
+
+const contactPoints = ['127.0.0.1']
+const localDataCenter = 'datacenter1'
+const keyspace = 'streamr_dev_v2'
+
+const HOST = '127.0.0.1'
+const API_URL = `http://${HOST}/api/v1`
+const WS_PORT = 12751
+const TRACKER_PORT = 17750
+const STORAGE_NODE_PORT = 17751
+const BROKER_PORT = 17752
+
+const addStreamToStorageNode = async (streamId, storageNodeAddress, client) => {
+    await fetch(`${API_URL}/streams/${encodeURIComponent(streamId)}/storageNodes`, {
+        body: JSON.stringify({
+            address: storageNodeAddress
+        }),
+        headers: {
+            // eslint-disable-next-line quote-props
+            'Authorization': 'Bearer ' + await client.session.getSessionToken(),
+            'Content-Type': 'application/json',
+        },
+        method: 'POST'
+    })
+}
+
+describe('StorageConfig', () => {
+    let cassandraClient
+    let tracker
+    let storageNode
+    let broker
+    let client
+    let stream
+    const publisherAccount = ethers.Wallet.createRandom()
+    const storageNodeAccount = ethers.Wallet.createRandom()
+    const brokerAccount = ethers.Wallet.createRandom()
+
+    beforeAll(async () => {
+        cassandraClient = new cassandra.Client({
+            contactPoints,
+            localDataCenter,
+            keyspace,
+        })
+    })
+
+    afterAll(() => {
+        cassandraClient.shutdown()
+    })
+
+    beforeEach(async () => {
+        tracker = await startTracker({
+            host: HOST,
+            port: TRACKER_PORT,
+            id: 'tracker'
+        })
+        storageNode = await startBroker({
+            name: 'storageNode',
+            privateKey: storageNodeAccount.privateKey,
+            networkPort: STORAGE_NODE_PORT,
+            trackerPort: TRACKER_PORT,
+            streamrUrl: `http://${HOST}`,
```

Comment:
It wouldn't have to hold necessary that host of E&E must be the same as the host these nodes are hosted on? Should `STREAMR_URL` then be its own constant?

Reply:
Fixed